ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/TestCompactionMetrics.java

```diff
@@ -444,6 +457,36 @@ public void testDBMetrics() throws Exception {
 
     Assert.assertEquals(1,
         Metrics.getOrCreateGauge(MetricsConstants.COMPACTION_STATUS_PREFIX + "txn_to_writeid").intValue());
+
+    start = System.currentTimeMillis() - 1000L;
```

Comment:
em, why do you need to subtract 1s here?

Reply:
leftovers from copy-paste. I removed it